src/Resources/ResourceManager/Implementation/Policy/GetAzurePolicyAssignment.cs

```diff
@@ -27,6 +27,9 @@ namespace Microsoft.Azure.Commands.ResourceManager.Cmdlets.Implementation
     /// <summary>
     /// Gets the policy assignment.
     /// </summary>
+    [CmdletOutputBreakingChangeWithVersion(
+        typeof(PsPolicyAssignment), deprecateByAzVersion: "11.0.0", deprecateByVersion: "6.9.2", DeprecatedOutputProperties = new[] { "Properties" },
```

Comment:
the current module version is 6.9.1, the next major version should be 7.0.0.

Reply:
Fixed, thanks.